frontend/ui/uimanager.lua

```diff
@@ -743,7 +747,7 @@ function UIManager:_checkTasks()
             -- task is pending to be executed right now. do it.
             -- NOTE: be careful that task.action() might modify
             -- _task_queue here. So need to avoid race condition
-            task.action()
+            task.action(task.context)
```

Comment:
In your context (uh), this 3rd argument is indeed a "context". But may be, generically, in UIManager, this 3rd argument passed as an arg to the callback could be just named `arg` or `args` (we have quite a few of that here and there).

Reply:
Right, I'll make it args or arguments just like in Event.